### ISSUE TITLE
--universe doesn't work with Intel MPI if hydra process manager is used

### ISSUE THREAD

**Issue by boegel** [hpcugent/vsc-mympirun#41]
`--universe` doesn't work with Intel MPI if hydra process manager is used (the default for recent impi versions), because it triggers passing `--ncpus` to `mpirun`, which is not recognised by hydra.

The problem manifests itself like:

```
2015-10-09 15:26:45,575 ERROR      mympirun.RunAsyncLoopStdout MainThread  _post_exitcode:
problem occured with cmd ['mpirun', '..., '--ncpus=16', ...]:
output [mpiexec@gligar02.gligar.os] match_arg (../../utils/args/args.c:254): unrecognized argument ncpus=16
[mpiexec@gligar02.gligar.os] HYDU_parse_array (../../utils/args/args.c:269): argument matching returned error
[mpiexec@gligar02.gligar.os] parse_args (../../ui/mpich/utils.c:4206): error parsing input array
```

To reproduce:

```
$ module load intel/2015b
$ module load scripts  # provides mympirun
$ mympirun --universe 7 --sched=local hostname
```

Workaround is to enable use of MPD process manager, rather than hydra (which is the default):

```
$ export I_MPI_PROCESS_MANAGER=mpd
```

**Comment by boegel** [comment]
@stdweird any idea how much effort it would be to fix this?

**Comment by stdweird** [comment]
not really. probably need to factor out the univesre stuff in a method, and have different behaviour for hydra

**Comment by boegel** [comment]
fixed in #100
